x-pack/plugins/alerting/server/routes/bulk_delete_rules.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { IRouter } from '@kbn/core/server';
+import { verifyAccessAndContext, handleDisabledApiKeysError } from './lib';
+import { ILicenseState } from '../lib';
+import { AlertingRequestHandlerContext, INTERNAL_BASE_ALERTING_API_PATH } from '../types';
+
+export const bulkDeleteRulesRoute = ({
+  router,
+  licenseState,
+}: {
+  router: IRouter<AlertingRequestHandlerContext>;
+  licenseState: ILicenseState;
+}) => {
+  router.patch(
```

Comment:
`patch` seems unusual here - I would have guessed we'd used `post` instead.  Why did we pick `patch`?

Reply:
Hi, @pmuellr!
It was hard choice. We basically use delete, because user can pass up to 1000 ids in payload.
So, when we chose, we follow this:
POST is always for creating a resource
and
PATCH is always for updating a resource.
So basically that. Updating is not completely like deleting :-), but we need to choose something.